apps/prairielearn/src/lib/externalGradingSocket.ts

```diff
@@ -88,18 +96,7 @@ export function connection(socket: Socket) {
       authorizedEdit: msg.authorized_edit,
       renderScorePanels: true,
     }).then(
-      (panels) => {
-        callback({
-          submission_id: msg.submission_id,
-          answerPanel: panels.answerPanel,
-          submissionPanel: panels.submissionPanel,
-          extraHeadersHtml: panels.extraHeadersHtml,
-          questionScorePanel: panels.questionScorePanel,
-          assessmentScorePanel: panels.assessmentScorePanel,
-          questionPanelFooter: panels.questionPanelFooter,
-          questionNavNextButton: panels.questionNavNextButton,
-        });
-      },
+      (panels) => callback(panels),
```

Comment:
Turns out this is also fixing a bug: the `scorePanel` was missing in this update. I tested in master, and indeed the score panels are not updated when the result of the external grader is retrieved, while this PR fixes that.